charts/incubator/snapcast/values.yaml

```
@@ -0,0 +1,241 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: saiyato/snapserver
+  # -- image tag
+  # @default -- chart.appVersion
+  tag: v0.26.0
```

```suggestion
  tag:
```
This inherits from the `Chart.yaml` so we prefer to leave this blank -  then you only have to maintain version once not twice.